js/table/actionBlock.ts

```diff
@@ -554,13 +555,7 @@ export class ActionBlock {
                 return;
             }
 
-            const gameApi = new Game(host);
-            if (value) {
-                gameApi.sitOut(this.tableView.tableId);
-            } else {
-                gameApi.comeBack(this.tableView.tableId);
-            }
-
+            tableManager.toggleSkipDeals(value);
```

Comment:
This call imply that call to server would be part of `ActionBlock` which prevent it from running in the test suite. Or we have to add some unnescessary mocking. You should add such subscription to the TableView utilize KO subscription. For example: `this.actionBlock.skipDeals.subscribe((value) => this.toggleSkipDeals(value))`.

In general you should view ActionBlock as dummy which just represent UI, and all actions done by TableView which control ActionBlock and subscribe to it notifications, to do actual work.